README.md

```diff
@@ -1,2 +1,17 @@
 # publish-cmsis-example
-GitHub action to publish CMSIS example to keil.arm.com
+
+This repository contains the GitHub action used to publish CMSIS example to [keil.arm.com](https://keil.arm.com)
+
+## Usage
+
+In the steps in your workflow file:
+```yaml
+- uses: Arm-Debug/solar-pack-project-importer@latest
+  with:
+    branch: # Branch to run on
+    project-file: # Project to import (required)
+    dry-run: # Whether to just complete a dry-run. This can be useful to verify a project is convertable and uploadable without actually making it available on keil.arm.com
```

Comment:
```suggestion
    dry-run: # Whether to just complete a dry-run. This can be useful to verify a project is publishable without actually making it available on keil.arm.com
```